Sources/KovaleeSurvey/KovaleeSurveyManagerImpl.swift

```diff
@@ -26,28 +32,52 @@ class KovaleeSurveyManagerImpl: SurveyManager, Manager {
         }
     }
 
-    func sendEvent(with name: String, andProperties properies: [String : String]) {
-        // TODO: - implement
+    func sendEvent(with name: String, andProperties properies: [String : Any]?) {
+        
+        if isPageView(name) {
```

Comment:
Move this logic in KovaleeFramework. 

In general KovaleeFramework holds all the logic, while this Wrappers are simply to pass things around